ehr/cosri/default.env

```diff
@@ -0,0 +1,18 @@
+# Default docker-compose environment file (.env)
+# https://docs.docker.com/compose/environment-variables/#the-env-file
+# environmental variables for interpolation in docker-compose YAML files
+
+
+# change to match your public IP address
+# If your address was 140.142.171.22, BASE_DOMAIN would be: 140-142-171-22.nip.io
+BASE_DOMAIN=INSERT-PUBLIC-IP-ADDRESS.nip.io
+
+
+# https://docs.docker.com/compose/reference/envvars/#compose_project_name
+# Containers started with the below value will have their names prefixed with it
+COMPOSE_PROJECT_NAME=cosri-dev
```

Comment:
as discussed in slack, i'd pull this paragraph in the name of simplicity, but i follow your reasons for inclusion.